### PR TITLE
Use better scoring/legend for GOATs

### PR DESCRIPTION
**Pull request by blakeparkinson**
this is pretty ugly code

### CODE REVIEW

File: public/javascripts/src/quiz.js [hunk 1]
```diff
@@ -104,49 +104,74 @@ var uploadScore = function(correct_answers){
...
-        label = '6-8';
+        label = '10-14';
         break;
       case 'mlow':
-        label = '3-5';
+        label = '5-9';
         break;
       case 'low':
-        label = '2 or Fewer';
+        label = '5 or Less';
```
Comment (kjgross) [inline public/javascripts/src/quiz.js]: 4 or Fewer/Less

File: public/javascripts/src/quiz.js [hunk 2]
```diff
@@ -228,10 +253,15 @@ var activeCopyButton = function(){
...
 
-function getSpanColor(score){
+function getSpanColor(score, possible_score){
+  var intScore = parseInt(possible_score);
   if (league == 'nfl'){
-    score = score / 5
-  }else{score = score/3 }
+    score = score / 5;
+  }
+  else if (intScore <= 10){
```
Comment (kjgross) [inline public/javascripts/src/quiz.js]: Don't we need to do this case first, for football goats/leaders which will only have 10?

**Comment by kjgross** [comment]
Quiz redirect not working for me again for pelicans or warriors.

thunder and goats and such seem to redirect just fine.

File: public/javascripts/src/quiz.js (hunk 2 above)
Comment (blakeparkinson) [inline public/javascripts/src/quiz.js]: ooo pretty smart

**Pull request merged**